controllers/delete.go

```diff
@@ -62,3 +62,26 @@ func (r *StorageSystemReconciler) deleteResources(instance *odfv1alpha1.StorageS
 
 	return fmt.Errorf("Waiting for %s %s to be deleted", instance.Spec.Kind, instance.Spec.Name)
 }
+
+func (r *StorageSystemReconciler) areAllStorageSystemsMarkedForDeletion() (bool, error) {
+
+	var storageSystems odfv1alpha1.StorageSystemList
+	err := r.Client.List(context.TODO(), &storageSystems)
+	if err != nil {
+		return false, err
+	}
+	// Check for number of StorageSystems in the cluster, If it is 2 or more that means StorageSystems will remain after deletion of current instance. If it is less than or equal to 1 then proceed with deleting all quickstarts
+	if len(storageSystems.Items) >= 1 {
```

Comment:
We dont need this if condition as We will go into loop only if we will have a items in the list

Reply:
done